connector/exceptionsconnector/README.md

```diff
@@ -0,0 +1,75 @@
+# Exceptions Connector
+
+| Status                   |               |
+| ------------------------ |---------------|
+| Stability                | [alpha] |
+| Supported pipeline types | See [Supported Pipeline Types](#supported-pipeline-types)  |
+| Distributions            | [contrib]     |
+
+## Supported Pipeline Types
+
+| [Exporter Pipeline Type] | [Receiver Pipeline Type] |
+| ------------------------ | ------------------------ |
+| traces                   | metrics, logs            |
+
+Extract metrics and logs from [recorded exceptions](https://opentelemetry.io/docs/reference/specification/trace/semantic_conventions/exceptions/) from spans.
+
+## Overview
+
+Each metric will have _at least_ the following dimensions because they are common across all spans:
+- Service name
+- Span kind
+- Status code
+- Exception message
+- Exception type
+
+This connector lets traces to continue through the pipeline unmodified.
```

Comment:
Connectors that derive signals from others should generally not also act as passthroughs for the original signal. I wrote up a lengthy rationale for this [here](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/18381#issuecomment-1424910925), but basically it's unnecessary and can introduce complications in some cases.

Reply:
Should I just remove this line?